cypress/integration/login.js

```diff
@@ -1,92 +1,118 @@
+const timeLoggingPage = new TimeLoggingPage();
+const common = new Common();
+    
+var roles = [
+    { name: 'User', 'tabCount': 1 }, 
+    { name: 'Team Lead', tabCount: 2 }, 
+    { name: 'Manager', tabCount: 5 }, 
+    { name: 'Accountant', tabCount: 5}, 
+    { name: 'Admin', tabCount: 6 }
+];
 
-    it('Should display validation for empty user after attempted loggin', function () {
-        
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')
-    })
+function makeId() {
+    let timeStamp = new Date().getTime();
+    return 'Random_Name_' + timeStamp;
+}
 
-    function testLogIn(role) {
-        cy.visit('/');
-        cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Marius Lastauskas"]').click();
-        cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="' + role + '"]').click();
-        cy.get('[type="submit"]').click();
-    }
+function testLogIn(role) {
+    loginPage.visit();
+    loginPage.getUserDropdown().click({force:true});
+    loginPage.getUserFromDropdown('Marius Lastauskas').click();
+    loginPage.getRoleDropdown().click({force:true});
+    loginPage.getRoleFromDropdown(role).click();
+    loginPage.getSubmitButton().click();
+}
 
+describe('Sourcebook test Clients', function() {
     it('Should be available for Admin to create new client', function () {
-        testLogIn('Admin');
+        cy.loginAs("Admin");
 
-        let organization = makeid(20);
-        cy.visit('/clients');
-        cy.get('button').contains('Create Client').click();
-        cy.get('[id="clientDetailsForm.organization"]').type(organization);
-        cy.get('[id="clientDetailsForm.contacts_firstName_0"]').type(makeid(20));
-        cy.get('[id="clientDetailsForm.contacts_lastName_0"]').type(makeid(20));
-        cy.get('[id="clientDetailsForm.contacts_email_0"]').type(makeid(20) + '@mail.com');
-        cy.get('[type="submit"]').click();
+        const organization = makeId();
+        const email = makeId() + '@mail.com';
+        clientsPage.visit();
+        clientsPage.getCreateClientButton().click();
+        clientCreatePage.getFormOrganization().type(organization);
+        clientCreatePage.getFormFirstName().type(makeId());
+        clientCreatePage.getFormLastName().type(makeId());
+        clientCreatePage.getFormEmail().type(email);
+        clientCreatePage.getSubmitButton().click();
         
-        cy.visit('/clients');
-        cy.get('[class="field--filter"]').first().find('input').type(organization);
-        cy.get('[class="ag-body-viewport-wrapper"').find('.ag-body-viewport').should('have.length', 1);
+        clientsPage.visit();
+        clientsPage.getOrganizationFilter().type(organization);
+        clientsPage.getFilteredList().should('have.length', 1);
     })
+})
 
+describe('Sourcebook test Tasks', function() {

```

Comment:
This is a test case to create task, but you are still in login.js file, would make sense to put it in another file